openapi.yaml

```diff
@@ -5159,9 +5233,12 @@ paths:
     post:
       x-linode-grant: read_write
       summary: Reboot Linode
-      description: >
+      description: |
         Reboots a Linode you have permission to modify. If any actions are currently running or
         queued, those actions must be completed first before you can initiate a reboot.
+
+        **Bare Metal Beta**
+        Reboots a Bare Metal Instance you have permission to modify.
```

Comment:
As above, this is the same as a normal Linode reboot, but you don't have the option to send in a `config_id`.